.eslintrc.js

```diff
@@ -25,37 +50,11 @@ module.exports = {
    */
   rules: {
     ...baseline.rules,
-    'import/prefer-default-export': ['off'],
+    // TODO move to @mui/monorepo, codebase is moving away from default exports
+    'import/prefer-default-export': 'off',
     // TODO move rule into the main repo once it has upgraded
-    '@typescript-eslint/return-await': ['off'],
-
-    'no-restricted-imports': [
-      'error',
-      {
-        paths: [
-          {
-            name: '@mui/icons-material',
-            message: 'Use @mui/icons-material/<Icon> instead.',
-          },
-          {
-            name: 'lodash-es',
-            importNames: Object.keys(lodash).filter((key) => !ALLOWED_LODASH_METHODS.has(key)),
-            message:
-              'Avoid kitchensink libraries like lodash-es. We prefer a slightly more verbose, but more universally understood javascript style',
-          },
-          {
-            name: 'react-query',
-            message: 'deprecated package, use @tanstack/react-query instead.',
-          },
-        ],
-        patterns: [
-          {
-            group: ['lodash-es/*'],
-            message: 'Use `import { debounce } from "lodash-es";` instead.',
-          },
-        ],
-      },
-    ],
+    '@typescript-eslint/return-await': 'off',
+    'no-restricted-imports': ['error', noRestrictedImports],
```

Comment:
moved to a variable, `noRestrictedImports`, to be able to extend it in specific paths.